src/projection_report.R

```diff
@@ -173,16 +206,23 @@ loadfonts(device = 'pdf')
         summarise(n = n())
       actuals <- rbind(actuals, data.frame(date = c(date), variable = c("actual"), value = c(counts[[1]])))
     }
+    
+    actuals <- bind_rows(actuals, cic_transpose %>% 
+                           select(date, as.character(test.age)) %>% 
+                           mutate(variable = "counts") %>% 
+                           rename(value = as.character(test.age)) %>% 
+                           mutate(value = as.numeric(as.character(value)))
+    )
+    
     actuals$date <- as.Date(actuals$date)
     print(ggplot() +
-            geom_line(data = actuals, aes(x = date, y = value)) +
+            geom_line(data = actuals, aes(x = date, y = value, colour = variable)) +
             geom_line(data = projected, aes(x = date, y = median), linetype = 2) +
             geom_ribbon(data = projected, aes(x = date, ymin = lower.ci, ymax = upper.ci), fill = "gray", alpha = 0.3) +
             geom_ribbon(data = projected, aes(x = date, ymin = q1, ymax = q3), fill = "gray", alpha = 0.3) +
-
             geom_vline(xintercept = train_from, color = "red", linetype = 2) +
             theme_mastodon +
-            scale_color_manual(values = colours) +
+            #scale_color_manual(values = colours) +
```

Comment:
Just checking: are the default colours really the ones we want?

Reply:
Ah right, thanks for flagging this @henrygarner and I don’t want to waste too much time on it, but to be able to plot the two “real” datasets I had to combine them (as you’ve seen) and create a column which differentiates between them. It was a problem between a discrete and a continuous datasets. Colouring only then worked by declared the column variable as where colours are dictated. To get this to work I had to comment out the scale_color_manual . So happy to use the non-default colours but I can’t figure out how to apply this